src/style.css

```diff
@@ -1,3 +1,90 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
+  list-style: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
You have repeated the font-family property for the body tag. This is a code quality issue because it's redundant and unnecessary. You already declared the font-family property in the html selector, which will be inherited by all child elements, including the body.